### PR TITLE
fix: collection notification integration

### PR DESCRIPTION
**Pull request by capJavert**
Integration fixes found when connecting on frontend and e2e testing:
- missing worker subscription
- missing worker registration in export
- notification icon and template
- notification unique key, using `metadataUpdatedAt` because without it `ID_notification_uniqueness` would fail and any notification after first one would not save to DB thus any collection updated after first one would be missed
- expose `numTotalAvatars` field for frontend

### CODE REVIEW

File: src/schema/notifications.ts [hunk 1]
```diff
@@ -137,6 +137,10 @@ export const typeDefs = /* GraphQL */ `
     Attachments of the notification
     """
     attachments: [NotificationAttachment!]
+    """
+    Total number of avatars
+    """
+    numTotalAvatars: Int
```
Comment (kopancek) [inline src/schema/notifications.ts:143]: This change seems unrelated to the fix to me. Can you explain why it's needed in this PR?

**Review by kopancek — COMMENTED** [review]
All looks good to me, just 1 thing I don't understand.

**Review by rebelchris — APPROVED** [review]
Looks good to me, especially liking the test addition we forgot the registration before

File: src/schema/notifications.ts (hunk 1 above)
Comment (capJavert) [reply src/schema/notifications.ts:143]: We have to add new field to Notification graphql type as well so that client can request it.

File: __tests__/workers/notifications/collectionUpdated.ts [hunk 2]
```diff
@@ -152,6 +155,15 @@ beforeEach(async () => {
...
 
 describe('collectionUpdated worker', () => {
+  it('should be registered', () => {
+    const worker = workers.find(
+      (item) =>
+        item.subscription === 'api.post-collection-updated-notification',
+    );
+
+    expect(worker).toBeDefined();
+  });
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply __tests__/workers/notifications/collectionUpdated.ts:165]: Very cool initiative!

**Review by sshanzel — APPROVED** [review]
Looks great! 🥳

**Review by idoshamun — APPROVED** [review]
Ome minor comment but looks great!

File: src/graphorm/index.ts [hunk 3]
```diff
@@ -447,6 +447,9 @@ const obj = new GraphORM({
           parentColumn: 'id',
         },
       },
+      numTotalAvatars: {
+        select: 'numTotalAvatars',
+      },
```
[the remark this answers is not in the document]
Comment (capJavert) [reply src/graphorm/index.ts:452]: Oh nice, sorted.
